clients/benchmarks/client.cpp

```diff
@@ -669,3 +419,286 @@ int main(int argc, char* argv[])
 
     return 0;
 }
+
```

Comment:
This function reads benchmarking binary data from a file, and runs the tests in sequence.